workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/DashboardEmptyTable.scss

```diff
@@ -4,7 +4,7 @@
 	border-radius: 10px;
 	display: flex;
 	flex-direction: column;
-	padding: 64px 468px;
+	padding: 64px 423px;
```

Comment:
@claricebrasil can you find a better solutions for this?

```suggestion
	
```

Reply:
Sure! I made the changes to fix the padding issue. Thank you!